README.md

```diff
@@ -1,2 +1,55 @@
 # gradle-crowdin-cli-plugin
 A gradle plugin to utilize the crowdin cli to interact with crowdin
+
+## Usage
+
+You need to copy the crowin-cli into you project and configure where to find the executable cli jar file (relative to your build file):
+
```

Comment:
Is it possible to ship the crodin cli jar with the plugin?

Reply:
Maybe, but for the first version I would like to make it like this.

Reply:
Advantage: The user can decide which cli version to use. In a perfect world it would be cool to configure the cli like
```
crowdin {
 cli = '2.0.1'
}
```
and download the binary file and put it into the source tree like the gradle wrapper is added to a project 🤔 
Maybe we can try to do that later

Reply:
Maybe we can just download the cli in general....